compiler/kernel/vexpr.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/brimdata/super/compiler/dag"
 	"github.com/brimdata/super/pkg/field"
 	"github.com/brimdata/super/runtime/sam/expr"
+	samexpr "github.com/brimdata/super/runtime/sam/expr"
```

Comment:
This package is already imported as expr by the previous line.

Reply:
Right